scripts/db_access/client_side_db.py

```diff
@@ -227,18 +244,41 @@ def _main(soledad, km, args):
             yield _export_key(args, km, args.expoert_public_key, private=False)
         if args.export_incoming_messages:
             yield _export_incoming_messages(soledad, args.export_incoming_messages)
-    except:
-        pass
+    except Exception as e:
+        logger.error(e)
     finally:
-        reactor.stop()
+        soledad.close()
+        reactor.callWhenRunning(reactor.stop)
```

Comment:
ah, I was looking on a quite short scope, now that I see the inlinecallbacks it make sense